app/rest/connector-generator/src/test/resources/swagger/petstore.param_connector.json

```diff
@@ -172,11 +172,11 @@
   },
   "actions": [
     {
-      "id": "io.syndesis:rest-swagger-connector:1.2-SNAPSHOT:_id_:updatePet",
+      "id": "io.syndesis:rest-swagger-connector:@syndesis-connectors.version@:_id_:updatePet",
```

Comment:
Would be good to do this is separate commit please - needed change but easier to track in separate commit as it's unrelated to core change of this PR.